ddl/index.go

```diff
@@ -698,6 +700,380 @@ func (w *worker) doBackfillIndexTaskInTxn(t table.Table, txn kv.Transaction, col
 	return taskRet
 }
 
+type addIndexWorker struct {
```

Comment:
Please add few comments for the following newly added code.

Reply:
added before `addTableIndex`.